### ISSUE TITLE
internal server error when inviting people

### ISSUE THREAD

**Issue by bbortt** [bbortt/event-planner#197]
```shell
2021-03-13T10:04:22.652851Zorg.springframework.dao.InvalidDataAccessApiUsageException: org.hibernate.TransientPropertyValueException: Not-null property references a transient value - transient instance must be saved before current operation : io.github.bbortt.event.planner.domain.Invitation.role -> io.github.bbortt.event.planner.domain.Role; nested exception is java.lang.IllegalStateException: org.hibernate.TransientPropertyValueException: Not-null property references a transient value - transient instance must be saved before current operation : io.github.bbortt.event.planner.domain.Invitation.role -> io.github.bbortt.event.planner.domain.Role at
org.springframework.orm.jpa.EntityManagerFactoryUtils.convertJpaAccessExceptionIfPossible(EntityManagerFactoryUtils.java:371) at
org.springframework.orm.jpa.vendor.HibernateJpaDialect.translateExceptionIfPossible(HibernateJpaDialect.java:257) at
org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.translateExceptionIfPossible(AbstractEntityManagerFactoryBean.java:528) at
org.springframework.dao.support.ChainedPersistenceExceptionTranslator.translateExceptionIfPossible(ChainedPersistenceExceptionTranslator.java:61) at
org.springframework.dao.support.DataAccessUtils.translateIfNecessary(DataAccessUtils.java:242) at
org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:153) at
org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) at
org.springframework.data.jpa.repository.support.CrudMethodMetadataPostProcessor$CrudMethodMetadataPopulatingMethodInterceptor.invoke(CrudMethodMetadataPostProcessor.java:178) at
org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) at
org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:95) at
org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) at
org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212) at
com.sun.proxy.$Proxy175.save(Unknown Source) at
io.github.bbortt.event.planner.service.InvitationService.save(InvitationService.java:50) at
io.github.bbortt.event.planner.service.InvitationService$$FastClassBySpringCGLIB$$53951deb.invoke(<generated>) at
org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) at
org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:771) at
org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) at
org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) at
org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:366) at
org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:118) at
org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) at
org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) at
org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:691) at
io.github.bbortt.event.planner.service.InvitationService$$EnhancerBySpringCGLIB$$7b7a6fb2.save(<generated>) at
io.github.bbortt.event.planner.web.rest.InvitationResource.createInvitation(InvitationResource.java:92) at
io.github.bbortt.event.planner.web.rest.InvitationResource$$FastClassBySpringCGLIB$$40f4e0cf.invoke(<generated>) at
...
```

![image](https://user-images.githubusercontent.com/12272901/111026834-ee644000-83ec-11eb-8f37-e886b2a32835.png)

**Comment by bbortt** [comment]
version [406](https://github.com/bbortt/event-planner/releases/tag/circle_406).

**Comment by bbortt** [comment]
closed within #200.
